examples/kubernetes/connectivity-check/Makefile

```diff
@@ -28,24 +30,28 @@ all: $(ALL_TARGETS)
 
 $(DEFAULT_OUT): $(SRC)
 	@echo '# Automatically generated by Makefile. DO NOT EDIT' > $(DEFAULT_OUT)
-	$(QUIET)$(CUE) dump > $(DEFAULT_OUT)
+	$(QUIET)$(CUE) dump >> $@
+
+$(IPv6_ONLY_OUT): $(SRC)
+	@echo '# Automatically generated by Makefile. DO NOT EDIT' > $(IPv6_ONLY_OUT)
+	$(QUIET)$(CUE) -t ipFamily=IPv6 dump >> $@
```

Comment:
for reviewer, there was a mixed usage of $@ and $(OUT_FILE) in this Makefile. As I am changing this file, I just change all to $@ for standardization.

> $@
> The file name of the target of the rule. If the target is an archive member, then ‘$@’ is the name of the archive file. In a pattern rule that has multiple targets (see Introduction to Pattern Rules), ‘$@’ is the name of whichever target caused the rule’s recipe to be run.